### ISSUE TITLE
:wheelchair: multiple WCAG2AA principle errors in index.html

### ISSUE THREAD

**Issue by SindreKjelsrud** [MarkusJohansen/ChimeraCSS#177]
**Describe the bug**
Multiple errors found in index.html from running [pa11y](https://pa11y.org/) in terminal:
 • Error: Duplicate id attribute value "email" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #email
   └── `<input type="email" id="email" name="email">`

 • Error: Presentational markup used that has become obsolete in HTML5.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.Strike
   ├── html > body > main > p:nth-child(26) > strike
   └── `<strike>striked</strike>`

 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #unimplemented-container > select
   └── `<select> <optgroup labe...</select>`

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #unimplemented-container > select
   └── `<select> <optgroup labe...</select>`

 • Error: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
   ├── #unimplemented-container > fieldset > input:nth-child(2)
   └── `<input type="text">`

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #unimplemented-container > fieldset > input:nth-child(2)
   └── `<input type="text">`

 • Error: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
   ├── #unimplemented-container > fieldset > input:nth-child(4)
   └── `<input type="text">`

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #unimplemented-container > fieldset > input:nth-child(4)
   └── `<input type="text">`

 • Error: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
   ├── #unimplemented-container > fieldset > input:nth-child(6)
   └── `<input type="text">`

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #unimplemented-container > fieldset > input:nth-child(6)
   └── `<input type="text">`

**To Reproduce**
1. Go to root directory
2. Use 'pa11y apps/demo/index.html' command
3. See error

**Expected behavior**
No WCAG2AA errors at all.

**Additional context**
Fixed in PR #176.
